packages/frontend-2/components/settings/workspaces/billing/PricingTable/Plan.vue

```diff
@@ -172,36 +200,50 @@ const isSelectable = computed(() => {
   // Allow upgrades to higher tier plans
   return canUpgradeToPlan.value
 })
-const toggleEnabled = computed(() => {
```

Comment:
There is a bit of a difference here, if there current interval is monthly, it should be enable (even if the selected state is yearly)